neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
```

Comment:
Actually, PIL does have the quality setting it seems, `im.save('<path>', format='JPEG', quality=100)` but that is another dependency, too. Not sure whether matplotlib is worse than PIL?